From 2e959238c9f220ec8ffece75441f9e1213c6f4c5 Mon Sep 17 00:00:00 2001
From: Qi Yadong <yadong.qi@intel.com>
Date: Mon, 26 Aug 2019 09:49:53 +0800
Subject: [PATCH 2/5] CFG: reconfigure to adapt server

Increase max cpu number to 256
Increase top of mem to 128G
Increase max number of VT-d enagines to 16

Change-Id: Id9defb323955ae530271ce22df7c8775b02127b0
Signed-off-by: Qi Yadong <yadong.qi@intel.com>
---
 product/board/cel_nuc.cfg | 4 ++--
 product/celadon_64.cfg    | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/product/board/cel_nuc.cfg b/product/board/cel_nuc.cfg
index db1c1d46..916c0945 100644
--- a/product/board/cel_nuc.cfg
+++ b/product/board/cel_nuc.cfg
@@ -1,7 +1,7 @@
 # basic info
 EVMM_CMPL_FLAGS += \
- -DMAX_CPU_NUM=8 \
- -DTOP_OF_MEM=0x800000000 \
+ -DMAX_CPU_NUM=256 \
+ -DTOP_OF_MEM=0x2000000000 \
  -DTSC_PER_MS=2200000 \
  -DLIB_MP_INIT \
  -DLIB_EFI_SERVICES \
diff --git a/product/celadon_64.cfg b/product/celadon_64.cfg
index 8344298b..11c9442b 100644
--- a/product/celadon_64.cfg
+++ b/product/celadon_64.cfg
@@ -17,7 +17,7 @@ EVMM_CMPL_FLAGS += \
  -DPACK_LK \
  -DMODULE_ACPI \
  -DMODULE_VTD \
- -DDMAR_MAX_ENGINE=4 \
+ -DDMAR_MAX_ENGINE=16 \
  -DSKIP_DMAR_GPU
 
 EVMM_CMPL_FLAGS += \
-- 
2.21.0

